src/RequestValidator/Cms/Event/CreateEventRequestHandler.php

```diff
@@ -0,0 +1,33 @@
+<?php declare(strict_types=1);
+
+namespace BalticRobo\Api\RequestValidator\Cms\Event;
+
+use BalticRobo\Api\Model\Cms\Event\EventDTO;
+use BalticRobo\Api\RequestValidator\RequestValidatorInterface;
+use Symfony\Component\Validator\Constraints as Assert;
+
+final class CreateEventRequestHandler implements RequestValidatorInterface
+{
+    public function getRules(): Assert\Collection
+    {
+        return new Assert\Collection([
+            'year' => [
+                new Assert\NotBlank(),
+            ],
+            'eventStartsAt' => [
+                new Assert\NotBlank(),
+            ],
+            'registrationStartsAt' => [
+                new Assert\NotBlank(),
+            ],
+            'registrationStopsAt' => [
+                new Assert\NotBlank(),
+            ],
+        ]);
```

Comment:
Oh, mate, you checks only that your request have values. You should check that `year` is numeric that from ~2000 to ~2100. Also check that dates are DateTime object.

You should read more about constraint in Symfony Validator docs: https://symfony.com/doc/current/reference/constraints.html